README.md

```diff
@@ -765,6 +766,9 @@ The export function converts the logical data types of the datacontract into the
 if a server is selected via the `--server` option (based on the `type` of that server). If no server is selected, the
 logical data types are exported.
 
+#### Spark
+
+The export function converts the data contract specification into a StructType Spark schema. 
```

Comment:
Can you add a link to the spark schema in the docs? So people can go to the according docs for that StructType?

Reply:
I added some details about Spark Data Types